src/components/molecules/NavBar/NavBar.scss

```diff
@@ -3,41 +3,49 @@
 $nav-container-width: 96px;
 $nav-container-height: 66px;
 $nav-btn-hover-background: #8e5ffc;
-$nav-schedule-btn-width-clicked: 30px;
 $nav-btn-border-radius: 10px;
+$nav-schedule-btn-padding: 7px 27px;
 $menu-item-size: $spacing--lg * 2;
 $logo-size: $spacing--xxl + $spacing--md;
 $volume-popover-width-with-sliders: 30em;
 $volume-popover-width-without-sliders: 15em;
-$simplified-view-button--height: 32px;
-$simplified-view-button--padding: 0 10px;
+$simple-view-button-height: 32px;
+$simple-view-button-padding: 7px 10px;
+$arrow-margin: 0 11px;
+$arrow-height: 8px;
+$arrow-width: 14px;
+$dropdown-item-width: 173px;
+$dropdown-item-padding: 15px 22px 0 22px;
```

Comment:
Can these be made to reflect `$spacing--` constants more closely? 

Few ideas:
- Even if those names aren't used `11px` can be `12px` for consistency with other parts of the system. 
- `173px` looks like an arbitrary number.  Is `170px` too odd? 🤷
- `15px` or `16px` which is also one of those `$spacing--` constants.

Reply:
fixed in 2bd2a15b2b1fb206e5f1fba98e2a743e59efc0ac